ttnn/cpp/ttnn/operations/moreh/moreh_norm/device/moreh_norm_h/kernels/moreh_norm_h_kernel.cpp

```diff
@@ -143,12 +139,25 @@ void MAIN {
         cb_pop_front(cb_xpowadd, onetile);
         cb_push_back(cb_xpowsum, onetile);
 
-        power_tile_to_cb(cb_xpowsum, cb_tmp0, cb_tmp1, cb_recip_p_decimal, cb_tmp2, cb_y, recip_p, recip_p_is_negative);
```

Comment:
It seems better to use a define or if statement to branch, rather than simply deleting it like this.

Reply:
the only case this wouldn't get deleted is when `dims.size ==1`, is that what you mean?

Reply:
> the only case this wouldn't get deleted is when `dims.size ==1`, is that what you mean?

I meant that it might be better not to modify the code related to the already implemented MorehNorm(where p is a number).

Reply:
I checked again and `power_tile_to_cb` has already been moved to `moreh_root` operation so it's no longer needed in `moreh_norm` kernel. I know it's a bit janky but it's working for now.